src/discordcr/client.cr

```diff
@@ -84,7 +84,7 @@ module Discord
     # The *properties* define what values are sent to Discord as analytics
     # properties. It's not recommended to change these from the default values,
     # but if you desire to do so, you can.
-    def initialize(@token : String, @client_id : UInt64? = nil,
+    def initialize(@token : String, @client_id : (UInt64 | Snowflake)? = nil,
```

Comment:
To be consistent with other type union styles, we've been writing out explicitly `UInt64 | Snowflake | Nil` if there's more than one other type.